api/resources/translations/messages-es.properties

```diff
@@ -1197,221 +1197,237 @@ report.pregnancy_visit.referral_follow_up_needed = Referencia de Seguimiento Nec
+settings.restore.action = Subir código de la aplicación
+settings.restore.description = Importar un nuevo archivo JSON y sobrescribir la configuración actual.
+settings.restore.title = Subir código de la aplicación
+setup.language.outgoing.subtitle = Seleccione el idioma en el que a sus coordinadores les gustaría recibir mensajes automáticos de CHT.
+setup.language.subtitle = Seleccione el idioma predeterminado que utilizarán los usuarios de CHT. Los usuarios pueden establecer preferencias individuales después de iniciar sesión.
 sms_message.message = Mensajes entrantes
-sms_received = Recibimos su mensaje, lo procesaremos pronto. Si intentaba mandar un formulario de texto, favor intentarlo de nuevo con un código de formulario.
+sms_received = Recibímos su mensaje, lo procesaremos pronto. Si intentaba mandar un formulario de texto, favor inténtelo de nuevo con un código de formulario.
 state.cleared = borrado
 state.delivered = entregado
 state.denied = negado
-state.duplicate =
+state.duplicate = duplicado
 state.failed = fallado
-state.forwarded-by-gateway = Reenviado por gateway
-state.forwarded-to-gateway = Reenviado a gateway
+state.forwarded-by-gateway = Reenviado por Gateway
+state.forwarded-to-gateway = Reenviado a Gateway
 state.muted = apagado
 state.pending = pendiente
 state.received = recibido
-state.received-by-gateway = Recibido de gateway
+state.received-by-gateway = Recibido por Gateway
 state.scheduled = programado
 state.sent = enviado
 status.review.correct = Verificado\: correcto
 status.review.errors = Verificado\: errores
-status.review.title = Revisión del Responsable
+status.review.title = Revisión del responsable
 status.review.unverified = No verificado
 status.sms.invalid = SMS inválido
-status.sms.title = Validez SMS
+status.sms.title = Validez del SMS
 status.sms.valid = SMS válido
-submit.icon =
+submit.icon = Subir icono
 sync.last_success = Última sincronización
 sync.now = Sincronizar ahora
+sync.retry = Reintentar
+sync.feedback.failure.unknown = Sincronización fallida. No puede conectarse.
 sync.status.in_progress = Sincronizando...
 sync.status.not_required = Todos los reportes sincronizados
 sync.status.required = Reportes a sincronizar
 sync.status.unknown = No es posible conectarse
-sys.empty = El mensaje esta en blanco.
-sys.facility_not_found = No se encontro a la unidad de salud.
-sys.form_not_found = Formulario no encontrado.
-sys.incorrect_type =
+sys.empty = El mensaje está en blanco.
+sys.facility_not_found = No se encontró el centro de salud.
+sys.form_not_found = Formulario '{{form}}' no encontrado.
+sys.incorrect_type = Tipo de campo incorrecto {{key}}, esperado {{expectedType}}.
 sys.missing_fields = Campo invalido o faltante\: {{fields}}.
-sys.recipient_not_found = No se encontro destinatario para el mensaje.
-targets.active_pnc.title =
+sys.recipient_not_found = No se encontró el destinatario para el mensaje.
+targets.active_pnc.title = Mujeres en periodo activo de PNC
```

Comment:
I'm thinking about this just now.  Perinatal Care, or "PNC" in Spanish, is called "Cuidado Perinatal" or "CP." Do you think we should use the Spanish abbreviation?

Reply:
Yes! I made the change in all occurrences